osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -65,14 +67,16 @@ private void load()
                 tailContainer = new Container<DrawableSliderTail> { RelativeSizeAxes = Axes.Both },
                 tickContainer = new Container<DrawableSliderTick> { RelativeSizeAxes = Axes.Both },
                 repeatContainer = new Container<DrawableSliderRepeat> { RelativeSizeAxes = Axes.Both },
+                headContainer = new Container<DrawableSliderHead> { RelativeSizeAxes = Axes.Both },
+                repeatArrowContainer = new Container { RelativeSizeAxes = Axes.Both },
+                headOverlayContainer = new Container { RelativeSizeAxes = Axes.Both },
```

Comment:
I really don't know about this. Containers come with an overhead, most of these seem unnecessary (adding two more takes me over the limit of not caring).